src/app/core/services/address/address.service.ts

```diff
@@ -1,26 +1,32 @@
 import { Injectable } from '@angular/core';
+import { Store, select } from '@ngrx/store';
 import { Observable } from 'rxjs';
 import { map, mapTo } from 'rxjs/operators';
 
 import { AddressMapper } from 'ish-core/models/address/address.mapper';
 import { Address } from 'ish-core/models/address/address.model';
 import { Link } from 'ish-core/models/link/link.model';
 import { ApiService, resolveLink, resolveLinks, unpackEnvelope } from 'ish-core/services/api/api.service';
+import { getCustomerRestResource } from 'ish-core/store/user';
 
 /**
  * The Address Service handles the interaction with the REST API concerning addresses.
  */
 @Injectable({ providedIn: 'root' })
 export class AddressService {
-  constructor(private apiService: ApiService) {}
+  private customerRestResource = 'customers';
+
+  constructor(private apiService: ApiService, store: Store) {
+    store.pipe(select(getCustomerRestResource)).subscribe(restResource => (this.customerRestResource = restResource));
+  }
```

Comment:
I think it would be better, if the UserService supplies a method for the customer endpoint and this method is then used within this class

Reply:
... also for the other services depending on the specific endpoint. :wink:

Reply:
I have to take this back. As AddressService depends on ApiService via `resolveLinks` nothing would be gained. :cry: I'll have to sleep on it once more.